src/components/Login/index.js

```diff
@@ -45,26 +47,12 @@ export const Login = ({ onChange, onSubmit }) => {
               onChange={onChange}
             />
           </div>
-          <div className="signin-buttons">
-            <button
-              type="submit"
-              className="btn btn-primary sign-in-bt"
-              id="signInBt"
-              name="signInBt"
-            >
-              Sign In
-            </button>
-            <div className="social-login-icons">
-              <small className="text-muted p-1">
-                Or Sign In with Social Media:
-                {' '}
-              </small>
-              <div className="icons-group text-center">
-                <SocialLogin />
-              </div>
-            </div>
+          {loginSigninButtons('signin-buttons', 'btn btn-primary sign-in-bt', 'signInBt', 'signInBt', 'Sign In')}
+          <div className="d-flex justify-content-around align-items-end">
+            {generateRedirectLinks('', 'passwordreset', 'Forgot password?')}
+            {generateRedirectLinks('', 'signup', 'Not yet a member?')}
           </div>
-          {generateRedirectLinks('Not yet a member? ', 'signup', 'Sign Up')}
+          {/* {generateRedirectLinks('Not yet a member? ', 'signup', 'Sign Up')} */}
```

Comment:
why did you comment this out?

Reply:
@RachaelNantale thanx for pointing this out.
This was duplicated so it needs to be removed which I've just done

Reply:
Your welcome